eva_submission/nextflow/accession.nf

```diff
@@ -86,7 +86,7 @@ if (is_human_study) {
     csi_vcfs = Channel.empty()
     Channel.fromPath(params.valid_vcfs)
         .splitCsv(header:true)
-        .map{row -> tuple(file(row.vcf_file), row.assembly_accession, file(row.fasta), file(row.report))}
+        .map{row -> tuple(file(row.vcf_file), row.assembly_accession, row.aggregation, file(row.fasta), file(row.report))}
```

Comment:
The `row` object is not using the position in the `csv` but the header name to find the column. 
I reordered them because it seemed sensible but I guess that's debatable that it is.

Reply:
Oh right true... and I was going to point out that you put it at the end in `variant_load.nf` but then realised the ordering of the tuple is not at all the same between these anyway... so it's probably fine :sweat_smile: :+1: